client/src/components/Workflow/Import/TrsSearch.vue

```diff
@@ -71,7 +78,7 @@ watch(query, async () => {
 function onTrsSelection(selection: TrsSelection) {
     trsSelection.value = selection;
     trsServer.value = selection.id;
-    query.value = "";
+    query.value = props.search ? props.search : "";
```

Comment:
this needs refinement, the search needs to wait until the stores are fetched, so the watcher cannot execute immediately when it gets query from props